vulnerabilities/severity_systems.py

```diff
@@ -49,6 +53,9 @@ def compute(self, scoring_elements: str) -> str:
         """
         return str(CVSS2(vector=scoring_elements).base_score)
 
+    def get(self, scoring_elements: str) -> dict:
```

Comment:
Please add tests for all the scenarios, also check if there may exist a case where it can fail.

Reply:
Done